service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/GeneratedOrderEventHandler.java

```diff
@@ -47,6 +54,26 @@ public void sendEmailsForOrder(final GeneratedOrderEvent orderEvent) {
             caseDetails);
     }
 
+    @EventListener
+    public void sendNotificationToAllocatedJudgeForOrder(final GeneratedOrderEvent orderEvent) {
+        final EventData eventData = new EventData(orderEvent);
+        CaseData caseData = mapper.convertValue(eventData.getCaseDetails().getData(), CaseData.class);
+
+        Map<String, Object> parameters = orderIssuedEmailContentProvider
+            .buildAllocatedJudgeOrderIssuedNotification(eventData.getCaseDetails());
+
+        String email = getAllocatedJudgeEmail(caseData);
+
+        notificationService.sendEmail(ORDER_ISSUED_NOTIFICATION_TEMPLATE_FOR_JUDGE, email, parameters,
+            eventData.getReference());
+    }
+
+    private String getAllocatedJudgeEmail(CaseData caseData) {
+        Optional<Element<GeneratedOrder>> generatedOrder = caseData.getOrderCollection()
+            .stream().reduce((first, last) -> last);
```

Comment:
Is it worth extracting lines 72-73 as a method on caseData? Both ways we should test this method. But we don't necessarily have to test line 74.

Reply:
done :)